### PR TITLE
Clarifying and url fix

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2491

Source of the note https://support.google.com/a/answer/7502379?visit_id=637496235003641124-1267497588&rd=1#zippy=%2Cadd-a-secondary-domain-manage-teams-at-different-domains

The url is replaced with the en-us path because that is the only link that works (language redir does not work for other languages)

@dariomws Kindly review

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -160,6 +160,8 @@ If your project doesn't already have all of the required APIs enabled, you must
 
    > [!NOTE]
    > A sub-domain of your primary domain is recommended. If another domain (such as "fabrikaminc.onmicrosoft.com") is set, Google will send emails to each individual address with a link to verify the permission to route mail. Migration won't complete until the verification is completed.
+   >
+   > The option to Add a domain will not be available if using the legacy free edition of G Suite.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:164]:
```suggestion
   > The option to add a domain will not be available if using the legacy free edition of G Suite. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by dariomws — APPROVED** [review]
Maybe it should be "The option Add a domain will not be available if using the legacy free edition of G Suite."?
And what about putting this note under the 2nd step?

**Comment by get-itips** [comment]
@JohanFreelancer9 take a look again please as I applied @dariomws suggestions.

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**